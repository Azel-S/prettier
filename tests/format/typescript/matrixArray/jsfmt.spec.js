run_spec(__dirname, ["typescript"], {matrixArray: false});
run_spec(__dirname, ["typescript"], {matrixArray: true});
