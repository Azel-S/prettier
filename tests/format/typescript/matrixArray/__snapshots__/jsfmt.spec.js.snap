// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matrixArray.ts - {"matrixArray":false} format 1`] = `
====================================options=====================================
matrixArray: false
parsers: ["typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
// Test file for array matrix formatting feature in TypeScript

// Single line array of numbers
const testArray9: number[] = [1, 2, 3, 4];

// Multiline array of numbers, already formatted
const testArray8: number[] = [
    1, 2,
    3, 4
];

// Multiline array of numbers, needs reformatting
const testArray7: number[] = [
    1, 2,
3, 4 
  // Intentionally misaligned for testing
];
=====================================output=====================================
// Test file for array matrix formatting feature in TypeScript

// Single line array of numbers
const testArray9: number[] = [1, 2, 3, 4];

// Multiline array of numbers, already formatted
const testArray8: number[] = [1, 2, 3, 4];

// Multiline array of numbers, needs reformatting
const testArray7: number[] = [
  1, 2, 3, 4,
  // Intentionally misaligned for testing
];

================================================================================
`;

exports[`matrixArray.ts - {"matrixArray":true} format 1`] = `
====================================options=====================================
matrixArray: true
parsers: ["typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
// Test file for array matrix formatting feature in TypeScript

// Single line array of numbers
const testArray9: number[] = [1, 2, 3, 4];

// Multiline array of numbers, already formatted
const testArray8: number[] = [
    1, 2,
    3, 4
];

// Multiline array of numbers, needs reformatting
const testArray7: number[] = [
    1, 2,
3, 4 
  // Intentionally misaligned for testing
];
=====================================output=====================================
// Test file for array matrix formatting feature in TypeScript

// Single line array of numbers
const testArray9: number[] = [1, 2, 3, 4];

// Multiline array of numbers, already formatted
const testArray8: number[] = [
  1, 2,
  3, 4,
];

// Multiline array of numbers, needs reformatting
const testArray7: number[] = [
  1, 2,
  3, 4,
  // Intentionally misaligned for testing
];

================================================================================
`;
